### PR TITLE
Example/dac/stm32f07

### PR DESCRIPTION
**Pull request by kuldeepdhaka**

### CODE REVIEW

File: examples/stm32/f0/stm32f072-discovery/dac-simple/dac-simple.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#include <libopencm3/cm3/nvic.h>
+
+#include <libopencm3/stm32/dac.h>
+#include <libopencm3/stm32/timer.h>
+
+void rcc_clock_setup_in_hsi_out_48mhz_corrected(void);
+void dac_init(void);
+void gpio_init(void);
+
+void rcc_clock_setup_in_hsi_out_48mhz_corrected(void)
```
Comment (karlp) [inline examples/stm32/f0/stm32f072-discovery/dac-simple/dac-simple.c]: is there a corresponding pull in the main library for this?

File: examples/stm32/f0/stm32f072-discovery/dac-simple/README [hunk 2]
```diff
@@ -0,0 +1,6 @@
+------------------------------------------------------------------------------
+README
+------------------------------------------------------------------------------
+
+Simple DAC Test example
```
Comment (karlp) [inline examples/stm32/f0/stm32f072-discovery/dac-simple/README]: Please expand on what this actually does.  Even better is to combine adc-dac-print sort of thing, so you can with a simple wire externally see both peripherals actually working.  needing a scope to see this working is problematic for many people.

File: examples/stm32/f0/stm32f072-discovery/dac-timer/dac-timer.c [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+{
+	rcc_periph_clock_enable(RCC_GPIOA);
+	gpio_mode_setup(GPIOA, GPIO_MODE_ANALOG, GPIO_PUPD_NONE, GPIO4 | GPIO5);
+}
+
+void tim_init(void)
+{	
+	rcc_periph_clock_enable(RCC_TIM6);
+	
+	//timer_reset(TIM6);
```
Comment (karlp) [inline examples/stm32/f0/stm32f072-discovery/dac-timer/dac-timer.c]: don't include commented out code in examples.

File: examples/stm32/f0/stm32f072-discovery/dac-timer/README [hunk 4]
```diff
@@ -0,0 +1,6 @@
+------------------------------------------------------------------------------
+README
+------------------------------------------------------------------------------
+
+DAC with timer to generate triangular waveform
```
Comment (karlp) [inline examples/stm32/f0/stm32f072-discovery/dac-timer/README]: Either this file, or the code itself, or both should be expanded with the frequency of the waveform, and how that was done.  These are meant to be useful examples, with useufl documentation.

**Comment by kuldeepdhaka** [comment]
no energy left to manage.

**Pull request closed without merging**